### DISCUSSION TITLE
Apollo prime ng with out individual module and routing

### DISCUSSION THREAD

**Discussion by Github743** [primefaces/community#1813] [PrimeNG] [answered]
I purchased the Apollo lay out https://apollo.primeng.org/#/

![image](https://github.com/primefaces/community/assets/3626755/5b34fa1c-1638-47c9-82f2-498e2afdc471)
I am trying to integrate in my existing application with out having individual module and routing is it possible to do so

**Comment by Github743** [comment]
Why this is not responsive? 
![image](https://github.com/primefaces/community/assets/3626755/f3681ad6-ce9d-489c-bfd3-1382d57ba007)

When I open responsive from mobile option in chrome this is what I see
![image](https://github.com/primefaces/community/assets/3626755/3171bb80-f4c0-4241-81f8-299d8398a8ef)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

There is a maximum width setting for the content to avoid stretching the content too much. You can remove it or modify it.

`src/assets/layout/styles/layout/_responsive.scss` line 101

```
@media screen and (min-width: 1960px) {
    .layout-content, .landing-wrapper {
        width: 1504px;
        margin-left: auto !important;
        margin-right: auto !important;
    }
    
    .layout-topbar {
        width: 1504px;
        margin-left: auto !important;
        margin-right: auto !important;
    }
}
```

**Reply by Github743** [reply]
Thanks I will try and let you know

**Comment by Github743** [comment]
Thanks it worked as expected after commenting out the lines
